galite-core/src/main/kotlin/org/kopi/galite/visual/VExecFailedException.kt

```diff
@@ -48,4 +48,8 @@ open class VExecFailedException : VException {
    * Constructs an exception with no message.
    */
   constructor() : super()
+
+  companion object {
+    private const  val serialVersionUID = 0L
```

Comment:
add
companion object {
private const val serialVersionUID = 0L
}
in VException class